<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClientDialog</class>
 <widget class="QDialog" name="AddClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Prénom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firstname">
           <property name="placeholderText">
            <string>Prenom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lastname">
           <property name="placeholderText">
            <string>Nom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Adresse :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="address">
           <property name="placeholderText">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Ville :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="city">
           <property name="placeholderText">
            <string>Ville</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Code postal :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="postalCode">
           <property name="placeholderText">
            <string>Code postal</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Jour de rendez-vous :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="appointmentDate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Durée du rendez-vous :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="appointmentLength">
           <property name="placeholderText">
            <string>Durée du rendez-vous</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Priorité :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="priority">
           <property name="placeholderText">
            <string>Priorité (5 à 1)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Commentaires :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="comment">
           <property name="placeholderText">
            <string>Commentaires</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Numéro de téléphone :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phoneNumber">
           <property name="placeholderText">
            <string>Numéro de téléphone</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
